### PR TITLE
PI-863 removed unused css dependencies

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

**Comment by jwineman** [comment]
LGTM

File: index.php [hunk 1]
```diff
@@ -8,15 +8,13 @@
 $dataStore = new CF\WordPress\DataStore($logger);
 $wordpressAPI = new CF\WordPress\WordPressAPI($dataStore);
 
-$pluginData = get_plugin_data(CLOUDFLARE_PLUGIN_DIR . 'cloudflare.php');
+$pluginData = get_plugin_data(CLOUDFLARE_PLUGIN_DIR.'cloudflare.php');
```
Comment (XhmikosR) [inline index.php]: This is an unwanted change.

**Comment by XhmikosR** [comment]
@thellimist: you should cherry pick https://github.com/cloudflare/Cloudflare-WordPress/pull/148 too in this branch.

**Comment by XhmikosR** [comment]
Or I can just merge this there if it's easier for you.

**Comment by thellimist** [comment]
I'm planning to do delete other css files when upgrading to new frontend. Keeping all changes in single PR with multiple commits.

File: index.php (hunk 1 above)
Comment (XhmikosR) [inline index.php]: But I also ran `composer format` and those changes were kept. Makes me wonder differs.

**Comment by XhmikosR** [comment]
Allright, I'll just rebase mine then, after this gets merged, since I have already made the changes.

**Pull request merged**